src/main/kotlin/no/nav/klage/search/service/ElasticsearchService.kt

```diff
@@ -260,9 +287,9 @@ open class ElasticsearchService(private val esKlagebehandlingRepository: EsKlage
 
         erTildeltSaksbehandler?.let {
             if (erTildeltSaksbehandler) {
-                baseQuery.must(erTildeltSaksbehandler())
+                baseQuery.must(beTildeltSaksbehandler())
```

Comment:
Jeg er enig i at det blir bedre å lese slik. Men lurer på om det hadde vært bedre å gjøre en extension function som f.eks. `baseQuery.mustBe(tildeltSaksbehandler())`. Tenker høyt :) Synes uansett dette ble bedre enn hva det var.

Reply:
Jeg tror forøvrig jeg er enig med deg altså! :)